### PR TITLE
Adding getAdbPath method

### PR DESCRIPTION
**Pull request by moizjv**

### CODE REVIEW

File: test/unit/adb-commands-specs.js [hunk 1]
```diff
@@ -460,4 +460,7 @@ describe('adb commands', () => {
     adb.isValidClass('some.package/some.package.Activity').index.should.equal(0);
     should.not.exist(adb.isValidClass('illegalPackage#/adsasd'));
   });
+  it('getAdbPath should correctly return adbPath', () => {
+    adb.getAdbPath().should.equal(adb.adb.path);
```
Comment (sebv) [inline test/unit/adb-commands-specs.js]: Would it be better to rename adb.adb to adb.executable?

Comment (moizjv) [inline test/unit/adb-commands-specs.js]: what do you mean? replacing `adb` object in ADB properties with `executables`?

Comment (sebv) [inline test/unit/adb-commands-specs.js]: Yeah change the name. It's strange `adb.adb`.

**Comment by jlipps** [comment]
makes sense :+1:

**Pull request merged**